src/components/shared/MarkerTooltipContents.js

```diff
@@ -480,6 +480,14 @@ function getMarkerDetails(
                         formatMicroseconds
                       )
                     : null}
+                  {nursery.groups_pretenured
```

Comment:
BTW, is it ok to not show it if the value is 0 ?

Reply:
ping @PaulBone :)

Reply:
Yeah, that's intended. It's going to be zero for most events and I'd prefer not to clutter the tooltip.